packages/core/template/portfolio/contents/projects.md

```diff
@@ -0,0 +1,61 @@
+# Projects
+
+<include src="project1.md"/>
+<include src="project2.md"/>
+
+### Other projects
+
+<div class="row row-cols-1 row-cols-md-3 g-3">
+  <div class="col">
+    <div class="card">
+        <h4 class="card-header">Project title</h4>
+        <div class="card-body">
+            This is another project. 
+        </div>
+        <svg xmlns="http://www.w3.org/2000/svg" class="d-block user-select-none" width="100%" height="200" aria-label="Placeholder: Image cap" focusable="false" role="img" preserveAspectRatio="xMidYMid slice" viewBox="0 0 318 180" style="font-size:1.125rem;text-anchor:middle"><rect width="100%" height="100%" fill="#868e96"></rect><text x="50%" y="50%" fill="#dee2e6" dy=".3em">Image cap</text></svg>
```

Comment:
Could we change this to something more interesting? Maybe even the markbind logo? This also seems to be a good place to show off markbind's image and pic component (either or)

Reply:
Same for the other image caps!

Reply:
Hi for sure! I've replaced the svg with an markbind image